### ISSUE TITLE
exp-lookit-survey does not display dropdown multi-select/checkbox questions correctly

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/ember-lookit-frameplayer#361]
# TL;DR

Survey frame doesn't produce a dropdown menu with multi-select checkboxes, as shown in the documentation: http://www.alpacajs.org/docs/fields/select.html

# Which frame or frames does it affect?

`exp-lookit-survey`

# Example

Something like this should work, but doesn't. Instead it just displays all of the options at once (instead of as a dropdown) and there are no check boxes next to each option. 

```json
{
    "frames": {
        "study-survey": {
            "kind": "exp-lookit-survey",
            "formSchema": {
                "schema": {
                    "enum": [
                        "vanilla",
                        "chocolate",
                        "coffee",
                        "strawberry",
                        "mint"
                    ]
                },
                "options": {
                    "label": "Ice cream",
                    "helper": "What flavor of ice cream do you prefer?",
                    "multiple": true,
                    "type": "select",
                    "noneLabel": "Pick a flavor of ice cream!"
                }
            }
        }
    },
    "sequence": [
        "study-survey"
    ]
}
```

Result of the above code:

![Screenshot 2024-01-29 at 4 49 49 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/7e20fa05-00f0-4e8e-aa7c-ef586ab8112c)

Expected result:

![Screenshot 2024-01-29 at 4 49 29 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/7c623e87-7441-4b5b-8ea2-3e02f570d300)

![Screenshot 2024-01-29 at 4 49 21 PM](https://github.com/lookit/ember-lookit-frameplayer/assets/9041788/483939e9-c248-4c6f-9441-6395aed2cbe6)
